### ISSUE TITLE
[SCREENREADER]: Headings should be nested appropriately for ease of navigation

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#1074]
Migrated from vets.gov issue [15570](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15570)

## Issue
Screen reader users overwhelmingly use headings as their [primary means of navigating pages](https://webaim.org/projects/screenreadersurvey7/#finding) quickly and easily. Our Pre-need burial form has an H1, an H4, then an H2 which can be confusing when screen reader users open their headings menu. The headings will be displayed out of order in this menu in relation to their placement in the source order. Screenshot of the current headings attached.

## Acceptance Criteria

As a screen reader user, I want to hear the headings menu read out headings in the same order as they are presented in code source. 

## Environment

* Win10
* IE11
* JAWS

## Steps to Recreate

1. Enter `https://staging.va.gov/burials-and-memorials/application/530/claimant-information` in browser
2. Inspect with Chrome Dev tools or other browser
3. Verify the <h4> that appears after the <h1> in the source order

## Possible Fixes (optional)

If the issue is one that can be fixed easily by changing HTML, CSS, or JavaScript markup, include the snippet in the issue description. If it is a longer fix or you do not have a ready solution, remove this section.

```html
<div
  aria-valuemax="6"
  aria-valuemin="1"
  aria-valuenow="1"
  aria-valuetext="Step 1 of 6: Claimant Information"
  class="nav-header nav-header-schemaform"
  role="progressbar"
  tabindex="-1">
  	<!-- Make this heading an H2 -->
	<h4>
    <span class="form-process-step current">1</span>
    <span class="form-process-total">of 6</span>
    Claimant Information
  </h4>
</div>
```

## WCAG or Vendor Guidance (optional)

* [Headings and Labels: Understanding SC 2.4.6](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html)

---
![screen shot 2018-12-06 at 3 47 55 pm](https://user-images.githubusercontent.com/934879/49615174-573cf200-f972-11e8-8eb1-7e119a7dd9b5.png)

**Comment by Mottie** [comment]
Fixed in https://github.com/department-of-veterans-affairs/vets-website/pull/12353 - Numerous updates have followed.

**Comment by Mottie** [comment]
Work has been done
